### ISSUE TITLE
(stoa0285a.stoa001.opp-lat1-2) convert files 

### ISSUE THREAD

**Issue by lcerrato** [PerseusDL/canonical-latinLit#206]
I have looked at these files and can get them to run in my local Nemo.

- Added info to `<div type="edition">`
- Converted paragraphs to `<div type="textpart" subtype="paragraph" n="0">`
- Fixed header and refsDecl accordingly 
      `<cRefPattern n="paragraph" matchPattern="(\w+)" replacementPattern="#xpath(/tei:TEI/tei:text/tei:body/tei:div/tei:div[@n='$1'])">`
     `<p>This pointer pattern extracts paragraph</p>`
     `</cRefPattern>`
- Made lang three letters instead of two.

Remaining issue:
- `<listWit>` is not EpiDoc compliant so not sure what to do with the witness list.

**Comment by PonteIneptique** [comment]
I really think we should have a new repo for file like that. latin-criticalEd ?

**Comment by PonteIneptique** [comment]
This way we turn it to TEI only

**Comment by lcerrato** [comment]
I agree. Just wanted to see what could be done per the email request. Have no plans to commit at present. 
We may have more forthcoming.

**Comment by lcerrato** [comment]
EpiDoc does refer to witnesses, etc.  but no examples for inline appcrit.

**Comment by lcerrato** [comment]
NB. I committed the branch for future reference. The original request was for this to be in Leipzig's repo.
